jubakit/wrapper.py

```diff
@@ -0,0 +1,183 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import numpy as np
+from sklearn.base import BaseEstimator, ClassifierMixin
+from .classifier import Classifier, Config, Dataset
+
+
+class BaseJubatusClassifier(BaseEstimator, ClassifierMixin):
+  """
+  scikit-learn Wrapper for Jubatus Classifiers.
+  """
+  
+  def __init__(self, n_iter=1, shuffle=False, softmax=False, embedded=True, seed=None):
+    """
+    Creates a base class for Jubatus Classifiers"
+    """
+    self.softmax = softmax
+    self.n_iter = n_iter
+    self.shuffle = shuffle
+    self.embedded = embedded
+    self.seed = seed
+    self.fitted_ = False
```

Comment:
Scikit-learn APIs follow the following naming scheme.
- argument parameters are expressed without underscore, i.e. `self.shuffled`.
- parameters created during executions are expressed with underscore, i.e. `self.classes_`.
http://scikit-learn.org/stable/modules/generated/sklearn.linear_model.LogisticRegression.html#sklearn.linear_model.LogisticRegression

Which do you think we have to obey the scikit-learn APIs naming schema or not?

Actually, I notice that we don't have to use `self.fitted_` parameter because if classifier is not trained, `self.classifier_` object becomes None, and I remove `self.fitted_` parameter.